plugins/confluence.go

```diff
@@ -46,25 +48,36 @@ func (p *ConfluencePlugin) GetAuthorizationHeader() string {
 	return lib.CreateBasicAuthCredentials(p)
 }
```

Comment:
New, simple, and easy to read implementation of the function
```suggestion
func isValidURL(urlStr string) bool {
    parsedURL, err := url.Parse(urlStr)
    return err == nil && parsedURL.Scheme == "https"
}
```